tests/test_openapi.py

```diff
@@ -343,6 +345,30 @@ def test_openapi_types_uuid():
     assert openapi_type == Schema(type="string", format="uuid")
 
 
+def test_openapi_types_anyurl():
+    type_converter = OpenApiTypeConverter()
+
+    def conformance_test(openapi_type: Schema, the_type: Type):
```

Comment:
@wouterdb I added your suggested changes, but I have to say that I don't understand what it does.  Could you explain a bit further?

I don't understand either where is the duplicated logic

Reply:
Also, the test now fails with this error:
```
>       assert openapi_type == pydantic_result
E       AssertionError: assert Schema(ref=None, title=None, required=None, type='string', items=None, properties=None, description=None, format='uri', default=None, nullable=None, readOnly=None, example=None, deprecated=None, anyOf=None, enum=None, additionalProperties=None) == SchemaBase(ref=None, title=None, required=None, type='string', items=None, properties=None, description=None, format='uri', default=None, nullable=None, readOnly=None, example=None, deprecated=None, anyOf=None, enum=None)
E         +Schema(ref=None, title=None, required=None, type='string', items=None, properties=None, description=None, format='uri', default=None, nullable=None, readOnly=None, example=None, deprecated=None, anyOf=None, enum=None, additionalProperties=None)
E         -SchemaBase(ref=None, title=None, required=None, type='string', items=None, properties=None, description=None, format='uri', default=None, nullable=None, readOnly=None, example=None, deprecated=None, anyOf=None, enum=None)
E         Full diff:
E         - SchemaBase(ref=None, title=None, required=None, type='string', items=None, properties=None, description=None, format='uri', default=None, nullable=None, readOnly=None, example=None, deprecated=None, anyOf=None, enum=None)
E         ?       ----
E         + Schema(ref=None, title=None, required=None, type='string', items=None, properties=None, description=None, format='uri', default=None, nullable=None, readOnly=None, example=None, deprecated=None, anyOf=None, enum=None, additionalProperties=None)
E         ?
```
But I don't know if it means my change created consistency issues or if the test is wrong?